.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,23 +11,34 @@ Thanks for stopping by to let us know something could be better!
 Please run down the following list and make sure you've tried the usual "quick fixes":
 
   - Search the issues already opened: https://github.com/googleapis/google-api-python-client/issues
-  - If you have a question, post on Stackoverflow under the `google-api` tag.
-  - If you are reporting an issue or requesting a feature for a G Suite API, please use their [public issue tracker](https://gsuite-developers.googleblog.com/2017/03/a-new-issue-tracker-for-g-suite.html)
+  - Search the issues on our "catch-all" repository: https://github.com/googleapis/google-cloud-python
```

Comment:
Is this still relevant? Or should we be more specific and point the users to the main repository's README, as the latter contains the links to new split repos that are more likely to be monitored by the corresponding library's maintainers?

Reply:
Will tweak the template to be more specific when possible.

Reply:
I deleted the bit with the google-cloud-python repo from the template. If they've already found this template, they've found their way to the issue tracker :)